layers/meta-balena-allwinner/conf/machine/bananapi-m1-plus.conf

```diff
@@ -6,6 +6,7 @@ MACHINEOVERRIDES = "bananapi:${MACHINE}"
 
 include conf/machine/bananapi.conf
 
+PREFERRED_PROVIDER_virtual/kernel = "linux"
+PREFERRED_VERSION_linux = "4.19.76%"
```

Comment:
I think we could remove this as it will anyway default to 4.19.76 right?

Reply:
Yes, it will default to 4.19.76. Will remove the PREFERRED_VERSION.